packages/react/src/components/FormLabel/FormLabel.js

```diff
@@ -9,14 +9,19 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import classnames from 'classnames';
 import { settings } from 'carbon-components';
+import resolveBaseTextDir from '../../tools/bidiUtils';
 
 const { prefix } = settings;
 
-const FormLabel = ({ className, children, id, ...other }) => {
+const FormLabel = ({ className, children, id, textDir, ...other }) => {
```

Comment:
Would it be possible for users to use the `dir` prop directly, e.g. 

```jsx
<FormLabel dir={resolveBaseTextDir(text)} />
```

Instead of adding in a `textDir` prop?

Reply:
OK I fixed this